lib/create-plugin.js

```diff
@@ -155,12 +159,24 @@ function createPlugin(instrumentationApi, config = {}) {
               instrumentationApi.setActiveSegment(resolverSegment)
 
               resolverSegment.name = `${RESOLVE_PREFIX}/${formattedPath}`
-
               resolverSegment.addAttribute(FIELD_PATH_ATTR, formattedPath)
               resolverSegment.addAttribute(FIELD_NAME_ATTR, info.fieldName)
               resolverSegment.addAttribute(RETURN_TYPE_ATTR, info.returnType.toString())
               resolverSegment.addAttribute(PARENT_TYPE_ATTR, info.parentType.toString())
 
+              if (instrumentationApi.isFunction(config.customResolverAttributes)) {
```

Comment:
So the main reason I'm doing this is that the docs say we should be getting `contextValue` from that object, but on some Apollo versions I tested it's still called `context`. In order to align with the docs, I decided to just destructure myself and always turn it into `contextValue`.

I'm also not sure if `willResolveField` will always get passed the same arguments as field resolvers. Right now it does, but we'd have to point our users to the plugin docs, and I've been trying to avoid that. They're not writing a plugin, so they shouldn't care to read about how it's written.

Do you strongly disagree? If so, we can do as you say and pass the object.

Reply:
I don't strongly disagree just disagree.  We can't predict signature changes apollo server will make in future versions. The docs state whatever is current, we support many different versions.  By destructuring we run the risk of key rename which you can already see is happening between context and contextValue.

Reply:
I think the docs are just wrong and this field is always called `context`, at least, using that passes all of the versioned tests. I'll submit a bug report to the Apollo docs.

Anyway, I've made the suggested change.